pkgs/by-name/gi/git-fixup/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+,
+}:
+let
+  inherit
+    (lib)
+    licenses
+    maintainers
+    ;
+
+  version = "0.1.8";
+  pname = "git-fixup";
```

Comment:
Why the separate binding for `pname`? It seems to only be used on 1 place.

Reply:
I think that's just the convention? :shrug:

Reply:
Just an oversight, I suppose.